src/main/java/com/github/boscojared/disparse/parser/reflection/Detector.java

```diff
@@ -38,24 +39,26 @@ public void reportMethodAnnotation(Class<? extends Annotation> annotation,
                                     CommandHandler handler =
                                             method.getAnnotation(CommandHandler.class);
                                     Command command = new Command(handler.commandName(),
-                                            handler.description(), handler.roles());
+                                                                  handler.description(), handler.roles());
```

Comment:
I feel like formatted with google-style this won't have such a large space here but I could be wrong

Reply:
right i used tweaked google style from another project, needs reformating